bundles/org.openhab.binding.unifiprotect/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="unifiprotect" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>UnifiProtect Binding</name>
+	<description>This is the binding for UnifiProtect.</description>
+	<connection>local</connection>
+	<service-id>org.openhab.unifiprotect</service-id>
+	<config-description-ref uri="binding:unifiprotect"/>
```

Comment:
Would it be possible to add a suggestion finder ? I'm not sure if they provide anything by mDNS or otherwise.